R/rendering.R

```diff
@@ -336,9 +336,79 @@ getCurveIDAnalsysiGroupResults <- function(curveids, ...) {
 #' curveData <- poIVPKCurveData$points
 #' plotCurve(curveData, params, paramNames = NA, outFile = NA, ymin = NA, logDose = FALSE, logResponse=TRUE, ymax = NA, xmin = NA, xmax = NA, height = 300, width = 300, showGrid = FALSE, showLegend = FALSE, showAxes = TRUE, plotMeans = FALSE, connectPoints = TRUE, drawCurve = FALSE, addShapes = TRUE, drawStdDevs = TRUE)
 #' 
+filterFlaggedPoints <- function(points, returnGood = TRUE) {
+  badFlag <- "knocked out"
+  if(returnGood) {
+    return(subset(points, userFlagStatus!=badFlag & preprocessFlagStatus!=badFlag & algorithmFlagStatus!=badFlag & tempFlagStatus!=badFlag))
+  } else {
+    return(subset(points, userFlagStatus==badFlag | preprocessFlagStatus==badFlag | algorithmFlagStatus==badFlag | tempFlagStatus==badFlag))
+  }
+}
+applyParsedParametersToFitData <- function(fitData, parsedParams, protocolDisplayValues) {
```

Comment:
Could you add a comment here as to what this new function does?

Reply:
I added a function doc string here.